### ISSUE TITLE
When I set the same data and environment and run test.py, why is yolov3 faster than yolov5?

### ISSUE THREAD

**Issue by LEEGILJUN** [ultralytics/yolov5#3246]
## ❔Question
The basic hyper parameters and environment were not changed. and same image
batch-size : 1
image size : 800x800
image : gray scaled Satellite image
Number of objects : 7
pretrained : coco2017 by yolov3 and v5
code : ultralytics pytorch code for v3 and v5

result
yolov3 : 40.7/1.4/42.2 ms inference/NMS/total
yolov5 : 71.8/1.2/73.0 ms inference/NMS/total

mAP
yolov3 : 0.707, 0.436
yolov5 : 0.785, 0.552

## Additional context
When I studied the yolo series, I think that v5 should be faster than v3 not only in performance but also in inference speed. And I did several experiments by k-folding the data currently in use. But the result was always yolov3 faster. Other kinds of data were the same when I did the experiment. I would like to know why this is happening.

**Comment by glenn-jocher** [comment]
@LEEGILJUN that's not correct. Our YOLOv3 README table has a direct comparison between YOLOv3 and YOLOv5, and YOLOv5 exceeds YOLOv3 in every metric, including speed, size, FLOPS, parameters, mAP, etc.
https://github.com/ultralytics/yolov3#pretrained-checkpoints

## Pretrained Checkpoints

[assets3]: https://github.com/ultralytics/yolov3/releases
[assets5]: https://github.com/ultralytics/yolov5/releases

Model |size<br><sup>(pixels) |mAP<sup>val<br>0.5:0.95 |mAP<sup>test<br>0.5:0.95 |mAP<sup>val<br>0.5 |Speed<br><sup>V100 (ms) | |params<br><sup>(M) |FLOPS<br><sup>640 (B)
---   |---                   |---                     |---                      |---                |---                     |---|---              |---
[YOLOv3-tiny][assets3] |640  |17.6     |17.6     |34.8     |**1.2** | |8.8   |13.2
[YOLOv3][assets3]      |640  |43.3     |43.3     |63.0     |4.1     | |61.9  |156.3
[YOLOv3-SPP][assets3]  |640  |44.3     |44.3     |64.6     |4.1     | |63.0  |157.1
| | | | | | || |
[YOLOv5l][assets5]     |640  |**48.2** |**48.2** |**66.9** |3.7     | |47.0  |115.4
